packages/api-explorer/src/components/SideNav/SideNav.spec.tsx

```diff
@@ -74,3 +80,35 @@ describe('SideNav', () => {
     )
   })
 })
+
+describe('Search', () => {
+  test('it filters methods and types on input', async () => {
+    renderWithSearchAndRouter(<SideNav api={api} specKey="3.1" />)
+    const searchPattern = 'embedsso'
+    const input = screen.getByLabelText('Search')
+    jest.spyOn(api, 'search')
+    await act(async () => {
+      /** Pasting to avoid triggering search multiple times */
```

Comment:
I don't understand the comment. Perhaps a little more clarity on why.  Possibly because entering characters being entered one by results in search being triggered for each character?

Something like

`Using ... triggers search multiple times. Use of paste avoids this`